terraform/common/s3.tf

```diff
@@ -16,6 +17,21 @@ resource aws_s3_bucket db_backups {
 resource aws_s3_bucket cloudfront_logs {
   bucket = "${data.aws_caller_identity.current.account_id}-tv-cloudfront-logs"
 
+  # ACL for awslogsdelivery from https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/AccessLogs.html#AccessLogsBucketAndFileOwnership
+  # Specifically setting the ACL grant from awslogsdelivery
+  # Because we are setting a single grant, we also need to add in the default for the bucket owner
+  grant {
+    id          = "c4c1ede66af53448b93c283ce9448c4ba468c9432aa01d700d3878632f77d2d0"
```

Comment:
Can you please move it to a local variable?

Reply:
Good spot. Have done that. Thanks.